scheduled-jobs/build/reposync/Jenkinsfile

```diff
@@ -13,6 +13,14 @@ b = build       job: '../aos-cd-builds/build%2Freposync', propagate: false,
                 parameters: [[$class: 'StringParameterValue', name: 'SYNC_VERSION', value: '4.0']]
 
 description += "${b.displayName} - ${b.result}\n"
+failed |= (b.result != "SUCCESS")
+
+b = build       job: '../aos-cd-builds/build%2Freposync', propagate: false,
+                parameters: [[$class: 'StringParameterValue', name: 'SYNC_VERSION', value: '4.1']]
+
+description += "${b.displayName} - ${b.result}\n"
+failed |= (b.result != "SUCCESS")
 
 currentBuild.description = description.trim()
-currentBuild.result = b.result
+currentBuild.result = failed?"FAILURE":"SUCCESS"
```

Comment:
Call me crazy, I like some space between my ternary args...

Reply:
Look... I copy pasted that from another job :P Blame who wrote that.
But I'll change it.